ui/packages/dashboard_client/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@pingcap-incubator/dashboard_client",
+  "version": "0.0.3",
+  "description": "",
+  "author": "pingcap-incubator",
+  "repository" : {
+    "type" : "git",
+    "url": "git@github.com:pingcap-incubator/tidb-dashboard.git"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "engines": {
+    "node": ">=8",
+    "npm": ">=5"
+  },
+  "scripts": {
+    "gen_api": "openapi-generator generate -i swagger.yaml -g typescript-axios -c .openapi_config.yaml -o src",
```

Comment:
I think this `package.json` needs `openapi-generator` as devDependency.

Reply:
done!